projects/opendr_ws/src/perception/scripts/landmark_based_facial_expression_recognition.py

```diff
@@ -53,6 +54,8 @@ def __init__(self, input_image_topic="/usb_cam/image_raw",
         """
 
         # Set up ROS topics and bridge
+        self.input_image_topic = input_image_topic
```

Comment:
The `self` variable should change in the `listen()` method below on line 94 (can't comment there).

```suggestion
        self.input_rgb_image_topic = input_rgb_image_topic
```